src/ppl/src/IOPlacer.cpp

```diff
@@ -1201,10 +1201,20 @@ void IOPlacer::placePin(odb::dbBTerm* bterm,
   Point lb = die_boundary.ll();
   Point ub = die_boundary.ur();
 
+  odb::dbTechLayer* tech_layer = tech_->findRoutingLayer(layer);
+  if (width < tech_layer->getWidth() ||
+      height < tech_layer->getWidth()) {
+    logger_->error(PPL,
+                   34,
+                   "Pin {} do not respect the min width {}u of layer {}",
```

Comment:
does not

I think it is good to include the problematic width in the message, e.g.

Pin {} has dimension {}u which is less than the min width {}u of layer {}.